python-interface/src/MiniBotScripts/SwarmMaster.py

```diff
@@ -19,11 +20,102 @@ def run(bot):
     mediateThread = Thread(target=z.mediate)
     mediateThread.start()
     threads.append(mediateThread)
-    # commands for bot movement itself
+    
+    #echobot(bot,z)
+    colorbot(bot,z)
+    
+def colorbot(bot,z):
```

Comment:
These are just helper functions we made for testing between echobot and colorbot. I'm assuming users can define their own helper functions in their custom scripts? I assumed it would be alright to do this since in the end colorbot is called in the run() function. Echobot was just our first step to reaching colorbot, but we didn't delete it for testing purposes. (Don't know if this answers your questions..?)